### ISSUE TITLE
[ Feature request ]

### ISSUE THREAD

**Issue by krishnareddy21** [alexjlockwood/ShapeShifter#290]
Can we have an option to view vector drawable code on console itself for instant working process coz when i work with 1000's of file i need to download/export each file and copy the required code from that file. It will be easy if u provide this option to copy code i want and do my work as fast i can.

thank you for this awesome tool.

**Comment by alexjlockwood** [comment]
Do you mean a batch SVG-to-VectorDrawable command line tool?

**Comment by krishnareddy21** [comment]
Not exactly, but what i mean an option to view xml code on webpage itself for the svg/drawae we import so that we can copy it and use instead of exporting exporting/downloading the xml file each time when we want to convert a file. 

And also that batch thing also good to add,we can upload multiple files at a time and download them at once.

**Comment by krishnareddy21** [comment]
just for more clear idea i am including this site here 
http://inloop.github.io/svg2android/
